postman/P4PA-Send-E2E.postman_collection.json

```
@@ -0,0 +1,143 @@
+{
+	"info": {
+		"_postman_id": "25b4e0d8-b3aa-4787-98fa-0e4e575f70a6",
+		"name": "P4PA-Send-E2E",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "36568841"
+	},
+	"item": [
+		{
+			"name": "postToken client-credentials Copy",
```

Rename this.
Moreover, if we introduce a folder structure, we could have a:
* 00.login folder to perform login and eventually other logged user required operation
* 0X.abc other folders on which you could configure on the folder itself the auth token and let the default behavior of all contained request to inherit the auth config
```suggestion
			"name": "postToken client-credentials",
```